From 059667d403af8e4c802eefc2f822b62150dbb5a5 Mon Sep 17 00:00:00 2001
From: Bharat B Panda <bharat.b.panda@intel.com>
Date: Tue, 14 Sep 2021 15:29:58 +0530
Subject: [PATCH] Update header config to allow older fw version

Removed older fw disallow configuration to allow driver to
fallback on older version of fw when required version not found.

Signed-off-by: Bharat B Panda <bharat.b.panda@intel.com>
---
 drivers/net/wireless/iwl7000/hdrs/config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/iwl7000/hdrs/config.h b/drivers/net/wireless/iwl7000/hdrs/config.h
index 4b4d684ad235..13f539e5de63 100644
--- a/drivers/net/wireless/iwl7000/hdrs/config.h
+++ b/drivers/net/wireless/iwl7000/hdrs/config.h
@@ -29,7 +29,7 @@
 #define CPTCFG_IWLWIFI_DEBUG 1
 #define CPTCFG_IWLWIFI_NUM_CHANNELS 2
 #define CPTCFG_IWLWIFI_SUPPORT_DEBUG_OVERRIDES 1
-#define CPTCFG_IWLWIFI_DISALLOW_OLDER_FW 1
+//#define CPTCFG_IWLWIFI_DISALLOW_OLDER_FW 1
 #define CPTCFG_IWLWIFI_NUM_STA_INTERFACES 1
 #define CPTCFG_REJECT_NONUPSTREAM_NL80211 1
 #define CPTCFG_IWLWIFI_ATLAS_PLATFORM_WORKAROUND 1
-- 
2.40.1

